mico-admin/src/app/api/api.service.ts

```diff
@@ -419,7 +419,7 @@ export class ApiService {
         return this.rest.post<any>(resource, null).pipe(map(val => {
 
             // TODO handle job ressource as soon as the api call returns a job ressource
-            return true;
+            return val;
```

Comment:
maybe use the same pattern as for other post requests here to properly address todo